src/tests/App.test.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import BottomNavigation from '../components/BottomNavigation';
+import App from '../App';
+import LandingPage from '../pages/LandingPage';
+import ImportPage from '../pages/ImportPage';
+import InfoPage from '../pages/InfoPage';
+
+let props;
+let initialTransactions;
+
+describe('<App />', () => {
+  beforeEach(() => {
+    props = {
+      saveStateToLocalStorage: jest.fn(),
+      hydrateStateWithLocalStorage: jest.fn()
+    };
+
+    initialTransactions = [
```

Comment:
In the future we might need a helper function that initalize transactions since it is something that is used in many components. I guess this will be developed naturally when we have more tests covered. But I would suggest to add a todo there at least.

Reply:
Yeah will do that